doc/changes/latest.inc

```diff
@@ -32,6 +32,7 @@ Enhancements
 - Add ``starting_affine`` keyword argument to :func:`mne.transforms.compute_volume_registration` to initialize an alignment with an affine (:gh:`11020` by `Alex Rockhill`_)
 - The ``trans`` parameter in :func:`mne.make_field_map` now accepts a :class:`~pathlib.Path` object, and uses standardised loading logic (:gh:`10784` by :newcontrib:`Andrew Quinn`)
 - Allow :func:`mne.beamformer.make_dics` to take ``pick_ori='vector'`` to compute vector source estimates (:gh:`19080` by `Alex Rockhill`_)
+- Add ``on_missing`` functionality to :class:`mne.channels.UpdateChannelsMixin` to tune drop_channels behaviour when specified channel names are not found(:gh:`11043` by `Andrew Quinn`_)
```

Comment:
```suggestion
- Add ``on_missing`` functionality to :class:`mne.channels.UpdateChannelsMixin` to tune drop_channels behaviour when specified channel names are not found (:gh:`11077` by `Andrew Quinn`_)
```

Reply:
Thanks @agramfort - now fixed in [9ef982d](https://github.com/mne-tools/mne-python/pull/11077/commits/9ef982d7edefed27a3648a5780d2e487e5bf7a3e)